packages/components/src/components.d.ts

```diff
@@ -119,9 +166,34 @@ declare namespace LocalJSX {
          */
         "scale"?: number | null;
     }
+    interface PostTabHeader {
+        /**
+          * The name of the panel controlled by the tab header.
+         */
+        "panel"?: string;
+    }
+    interface PostTabPanel {
+        /**
+          * The name of the panel, used to associate it with a tab header.
+         */
+        "name"?: string;
+    }
+    interface PostTabs {
+        /**
+          * The name of the panel that is initially shown.
+         */
+        "activePanel"?: string;
+        /**
+          * An event emitted whenever a new tab becomes active. The payload is the name of the associated panel.
+         */
+        "onTabChange"?: (event: PostTabsCustomEvent<string>) => void;
```

Comment:
```suggestion
        "onTabChange"?: (event: PostTabsCustomEvent<PostTabPanel["name"]>) => void;
```
Nitpick. I'm not sure that it will work with the optional character.

Reply:
This file is generated.
To update the declaration, we should update the component:
```
/**
 * An event emitted whenever a new tab becomes active.
 * The payload is the name of the associated panel.
 */
@Event() tabChange: EventEmitter<HTMLPostTabPanelElement['name']>;
```

Is that what you had in mind?
If so, would you change it everywhere a `string` is used to type a panel name variable (i.g. in the `show` method or for the `panel` prop of the tab header)?

Reply:
I would say if possible. And just to clarify, it's only a proposal, I know that it can be annoying in some situations, but it helps to read the code.

Reply:
I used `HTMLPostTabPanelElement['name']` in most place